### DISCUSSION TITLE
Literature Search & Writing Process - 2/27 Lab Meeting Topic

### DISCUSSION THREAD

**Discussion by ChrisMantegna** [RobertsLab/resources#1588] [Show and tell]
The focus this week is on the literature review and writing process. We all know the basic flow of a literature review, but keeping track and trying to sync multiple apps can lead to more time managing databases than synthesizing content. Please come to lab meeting on Monday prepared to talk about your processes, show any method/ tracker you use, and contribute any tips that help you in your searches and your writing.

Questions to jog your thoughts
*Literature Review*
* Does your process mirror the tried and true method?
* Tried & True Method
** Question
** Set scope of literature search (date range, specific aspect, keywords, etc.)
** Search & Read
** Revise question
** Read some more
* How do you catalog/ track your papers?
* What style of annotation and note taking works best for immediate and/ or long term recall (in your reference manager, not memory explicitly)?
* Do you keep track of your keyword searches?
* Is an annotated bibliography really the way to go?
* When do you decide you have enough literature?

*Writing Process*
* Do you begin writing after your initial analysis, write in small bits throughout, or wait until you’ve completed analysis and visualization?
* What is your writing process (assuming completed analysis and visualization)?
* Do you write methods first, get inspired by the literature and write a killer introduction, make an outline, etc.?
* What makes a good collaborative writing partner(s)?
* What are your biggest stumbling blocks and how do you get around them?
* What is your weakest writing ability? Your strongest?
* Do you write daily?

**Comment by sr320** [comment]
also @ https://sr320.github.io/lit-review/

## Literature Review
Does your process mirror the tried and true method?

**No**

```
Tried & True Method
Question
Set scope of literature search (date range, specific aspect, keywords, etc.)
Search & Read
Revise question
Read some more
```

How do you catalog/ track your papers?

**Paperpile**

What style of annotation and note taking works best for immediate and/ or long term recall (in your reference manager, not memory explicitly)?

**Only annotation is markup of PDFs.**

Do you keep track of your keyword searches?

**No**

Is an annotated bibliography really the way to go?

**Not really sure what this is**

When do you decide you have enough literature?

**When it feels be search is exhaustive**

## Writing Process
Do you begin writing after your initial analysis, write in small bits throughout, or wait until you’ve completed analysis and visualization?

**Start writing as soon as start methods**

What is your writing process (assuming completed analysis and visualization)?

**? see below**

Do you write methods first, get inspired by the literature and write a killer introduction, make an outline, etc.?

**Methods, Results, Discussion, Introduction, Abstract**

What makes a good collaborative writing partner(s)?

**Regularity in Google Docs, using comments, suggestions, and letter comment be closed by one who writes if not super, super clear.**

What are your biggest stumbling blocks and how do you get around them?

**Writing... Blocking time and just writing, not pretty, flow of ideas, then go back and edit**

What is your weakest writing ability? Your strongest?

**typos, spelling,  .... methods, and selling just below overselling**

Do you write daily?

**Manuscripts?, no**

**Comment by AHuffmyer** [comment]
# LITERATURE REVIEW 

## **Does your process mirror the tried and true method?** 

Nope! I tend not to follow a specific system with reading papers (and I could definitely learn to be more efficient). I tend to find papers of interest through Twitter or Google Scholar recommendations. If I have a specific manuscript or topic I am working on, then I will use a more structured keyword search approach. 

## **How do you catalog/ track your papers? What style of annotation and note taking works best for immediate and/ or long term recall (in your reference manager, not memory explicitly)?** 

I have been using Google Scholar reading lists to find and mark papers to read. I created a few lists in my Google Scholar for "Need to read" and "Completed". Then, I can add papers to my "need to read" list and once they are read I move them to my "completed" list. This has helped me keep track of the papers that are on my reading list. I have a couple of days each week where I block out an hour or two on my calendar to go through this list and read a few of the papers. 

I then try to keep track of reading notes for literature that I find on Twitter/Google Scholar outside of specific literature reviews for a manuscript. For example, [here is a reading notes document](https://docs.google.com/spreadsheets/d/1KjitP57Dk4kN6DRM2oK85sIV5Z2VIKto9WS_gDGqWM4/edit#gid=0) with papers I have read from January/February. I try to include keywords and projects that relate to each paper so that I can go back and search these documents when I am looking for relevant information for a manuscript. 

I have also recently used some AI tools to help with literature searches! A few of my favorites are [Elicit](https://elicit.org/), [Scite](https://scite.ai/), [Connected Papers](https://www.connectedpapers.com/), and [Inciteful](https://inciteful.xyz/), all of which have similar functions for finding and distilling papers. Elicit is definitely my go-to for finding and summarizing literature for a specific question or helping me shape questions. 

To catalog and cite papers, I use PaperPile! I use Google Docs for all my writing, so this works really nicely. 

## **Do you keep track of your keyword searches?** 

Not in a structured way... I can look at the keywords from papers in my reading notes document to help find related papers. 

## **Is an annotated bibliography really the way to go?**  

I think this depends on each person and how you like to store and track papers! PaperPile doesn't have great functions for annotating and organizing papers, so I don't have a great system for grouping/annotating in my citation list. But I find that the AI tools listed above help me find related literature. 

## **When do you decide you have enough literature?**  

I don't think I ever feel like I have "enough"! Some of the AI tools above help me find the most relevant related work so that I make sure I am reading the most important papers for a specific topic/question. Whether or not I have enough literature might depend on the type of paper I'm writing or what I'm working on (e.g., review vs proposal vs data manuscript). 

# WRITING 

## **Do you begin writing after your initial analysis, write in small bits throughout, or wait until you’ve completed analysis and visualization?** 

I try to write bits throughout. Typically, the first thing I write is the methods and then make an outline for what I expect the main results sections will be. Once I feel confident in the main results, I'll start writing the results section and then edit as I go with a more detailed analysis. 

## **What is your writing process (assuming completed analysis and visualization)? Do you write methods first, get inspired by the literature and write a killer introduction, make an outline, etc.?**  

It seems like my approach changes with each paper I do! Generally, I start with the methods, preferably during or right after I do an experiment. Next, I'll add the main results and figures. After I find the main story from the results, I will then outline the discussion section. I like to start by writing topic sentences that will form the first sentence of each paragraph. Then, I will assemble a list of papers and specific information that I want to cite from previous work and add supporting sentences to each topic sentence. Typically the order and structure of the discussion changes a lot as I work on the manuscript, but I find that starting with topic sentences with the main points I want to make is a good way to start. After I shape the outline and main content of the discussion, then I will go back to the introduction. This helps me identify the information that the reader needs to know and the main knowledge gaps that are addressed in the study. The abstract is almost always last! 

## **What makes a good collaborative writing partner(s)?** 

I love writing papers with a buddy! It can be so helpful to talk through ideas and brainstorm ways to make impactful and concise points. I particularly like to trade text back and forth. For example, we work together to write a list of topic sentences that structure our discussion/intro/results. Then, each of us takes one of those sentences and writes a short paragraph to describe and provide evidence for that statement on a timer (say 15-20 min). We then trade sections and edit and add to each other's work. At the end of the hour, we have 2-4 paragraphs that we have written together! It's so much easier to go through reiterative edits rather than writing alone from scratch. A good writing buddy is supportive but also critical and challenges you to become a better writer! 

## **What are your biggest stumbling blocks and how do you get around them?** 

Sometimes I really get stuck on making a sentence sound clear and concise or I have a messy idea that I need to shape into something more impactful. I find having a coauthor or collaborator take a look and offer some edits to be really helpful! I work much better with editing text than start writing from scratch. I have learned to take the approach to write something messy and then edit from there. Actually, ChatGPT has been amazingly helpful in this step! If I have a messy sentence or paragraph and I am stuck trying to make edits, I type "Please edit the following text to be more clear and concise" into ChatGPT. It will then generate edited text that provides a great starting point for further edits! It is especially helpful in writing discussions and abstracts. I recommend trying it out! I also use Grammarly and PaperPal as editing resources. 

## **What is your weakest writing ability? Your strongest?** 

I definitely struggle with making statements that are confident and impactful. I am pretty critical of my own results, so I often feel the need to qualify my results statements with phrases such as, "could suggest", "may imply", and "could have an impact", rather than saying "X increases Y" or "X leads to Y" more directly. Especially when submitting to higher impact journals this has been something I am working on! One of my strengths is being thorough in providing evidence for claims and pointing out where our results and literature contradict each other. 

## **Do you write daily?** 

Yes! I didn't use to write daily, but now I do. Even if I write one sentence, it's more progress than I would make only writing once a week or a few times a month!

**Reply by kubu4** [reply]
### Literature Review

> Does your process mirror the tried and true method?

No.

> How do you catalog/ track your papers?

Paperpile.

> What style of annotation and note taking works best for immediate and/ or long term recall (in your reference manager, not memory explicitly)?

Don't really do any note taking.

> Do you keep track of your keyword searches?

No.

> Is an annotated bibliography really the way to go?

Like Steven, I'm not familiar with what this is referring to.

> When do you decide you have enough literature?

For a manuscript/thesis - When it seems like all statements requiring references have references associated with them.

### Writing Process

> Do you begin writing after your initial analysis, write in small bits throughout, or wait until you’ve completed analysis and visualization?

- Start writing Methods to help get juices flowing, since they're usually straight forward and easy to complete.

What is your writing process (assuming completed analysis and visualization)?

- Start writing Methods to help get juices flowing, since they're usually straight forward and easy to complete.

- Write Results, since these can start out "simple" , as it's describing what the outcomes of an experiment were and doesn't require interpretation (i.e. brain power).

> Do you write methods first, get inspired by the literature and write a killer introduction, make an outline, etc.?

I was "taught" to write papers in the following order:

1. Material/Methods
2. Results
3. Introduction
4. Discussion
5. Abstract

The rationale being that these are in order of "easiest" to "hardest" (i.e. the amount of brain power needed). The Abstract may not be the "hardest", but since the abstract has length limitations _and_ usually has aspects of the 4 previous sections, it should be done last.

> What makes a good collaborative writing partner(s)?

Can't say I've ever done anything like this, beyond contributing to a shared document.


> What are your biggest stumbling blocks and how do you get around them?

Reading papers and writing papers. I really dislike all aspects of both things. In fact, it was this realization during grad school that led me to quitting my Ph.D. and switching to a M.S. degree instead. The prospect of being a P.I. and spending so much time reading/writing was overwhelming.

Admittedly, I haven't addressed the stumbling blocks. I just avoid doing these activities as much as possible. 

> What is your weakest writing ability? Your strongest?

Weakest: Motivation.

Strongest: Editorial.

> Do you write daily?

No.
